Unix/http/http.c

```diff
@@ -278,11 +278,11 @@ static MI_Result _Sock_ReadAux(
             handler->acceptDone = MI_TRUE;
             return _Sock_ReadAux(handler,buf,buf_size,sizeRead);
         }
-	else
+    else
         {
-	    SSL_load_error_strings(); // registers the error strings for all libcrypto and libssl functions.
-	    SSL_get_error(handler->ssl, res);
-	}
+        SSL_load_error_strings(); // registers the error strings for all libcrypto and libssl functions.
+        SSL_get_error(handler->ssl, res);
+    }
```

Comment:
If tabs are removed from this file, why is the indentation weird? See 282 and 283, above.